lib/events.js

```diff
@@ -479,23 +479,19 @@ EventEmitter.prototype.removeAllListeners =
     };
 
 EventEmitter.prototype.listeners = function listeners(type) {
-  var evlistener;
-  var ret;
-  var events = this._events;
+  const events = this._events;
 
-  if (events === undefined)
-    ret = [];
+  if (!events)
```

Comment:
Why this was changed back to `!events`?

Reply:
Complete accident. Will push a fix shortly.